free-1.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I am enthralled by this idea because it addresses a very relevant problem at Mudd. It seems like it would work well with a language like Prolog which I am interested in exploring more in depth, and is easily scalable so that if I have enough time I can add many additional features, but I should be able to get the base of the language created well within the alloted time frame. 
+
+I came up with this because I was interested in creating a Prolog like language that does some sort of pattern matching, rules, and data storage, I liked the ideas in the tree-tracer lanugage created by ellenfkh which stored relationships from complicated book series and allowed you to look up characters again later and I discussed my idea with Prof Ben and he gave me the idea of applying it to the specific domain of clinic team formation.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Optimizing clinic team formation choices.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would interact with the language by creating new students in the system with a set of things the student want, weighted by how important each is to him or her. The user can then type a command to get the best clinic team divisions. A program would look like instantiating a set of students and then typing a command to get the top few possible team divisions. This is the right way to interact with the domain because it is very similar to what is done now excpet that rather than having a person do it, the computer will be considering all the combinations of possible teams based on the criteria and choosing the best.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+If I create this as an internal DSL in Prolog, when a program runs it will run as a Prolog program, checking to see what combinations of variables match all the rules its been given and will then ouput the desired number of most optimal team divisions. If there are multiple "best" team divisions the program should let the user know about the tied team divisions. If there is no best division the program should print an error asking for more criteria.
 
```

Comment:
Prolog is definitely good for pattern matching, but will it provide the kind of user experience that you want? What languages do you think the clinic directors prefer to work with? Or do you think that because of Prolog's problem matching strength overweighs that?